src/files.ts

```diff
@@ -43,6 +44,30 @@ export interface FileX {
      * @returns An absolute file path to the downloaded/copied file
      */
     download(path?: string): Promise<string>;
+    /**
+     * This method will fetch the file URL and return an async iterator which
+     * yields every time a new chunk of data is read.
+     *
+     * If the `file_path` of this file object is `undefined`, this method will
+     * throw an error.
+     *
+     * @example
+     * ```ts
+     *  bot.on([":video", ":animation"], async (ctx) => {
+     *      // Prepare file for download
+     *      const file = await ctx.getFile();
+     *      // Start streaming file
+     *      const data = file.stream();
+     *      // Print the size of each chunk
+     *      for await (const chunk of data) {
+     *        console.log(`Read ${chunk.length} bytes`);
+     *      }
+     *  });
+     * ```
+     *
+     * @returns An iterator for the received data
+     */
+    getIterator(): AsyncGenerator<Uint8Array>;
```

Comment:
```suggestion
    stream(): AsyncGenerator<Uint8Array>;
```
is the better name so let's align the code with the docs

Reply:
```suggestion
    [Symbol.asyncIterator](): AsyncIterator<Uint8Array>;
```

Reply:
Or
```suggestion
    stream(): ReadableStream<Uint8Array>;
```